client/flutter/lib/generated/l10n.dart

```diff
@@ -517,7 +517,7 @@ class S {
 
   String get travelAdviceContainerText {
     return Intl.message(
-      'WHO continues to advise against the application of travel or trade restrictions to countries experiencing COVID-19 outbreaks. It is prudent for travellers who are sick to delay or avoid travel to affected areas, in particular for elderly travellers and people with chronic diseases or underlying healh conditions. “Affected areas” are considered those countries, provinces, territories or cities experiencing ongoing transmission of COVID-19, in contract to areas reporting only imported cases.',
+      'WHO continues to *advise against the application of travel or trade restrictions* to countries experiencing COVID-19 outbreaks.\n\n*It is prudent for travellers who are sick to delay or avoid travel to affected areas*, in particular for elderly travellers and people with chronic diseases or underlying healh conditions. “Affected areas” are considered those countries, provinces, territories or cities experiencing ongoing transmission of COVID-19, in contract to areas reporting only imported cases.',
```

Comment:
This file is autogenerated and translation should be added through the translation tool https://weblate.whoapp.org/